### PR TITLE
Show numerus sensitive message in comparison output when scopes are identical 

### PR DESCRIPTION
**Pull request by keberle94**
Superseeds #955

### CODE REVIEW

File: lib/compare_task.rb [hunk 1]
```diff
@@ -94,9 +97,13 @@ def render_comparison(description1, description2, scopes, options = {})
         identical = false if description1[scope] || description2[scope]
       end
     end
+    plural = "scopes are"
+    singular = "scope is"
+    count = identical_scopes.count
```
Comment (thardeck) [inline lib/compare_task.rb]: Why use variables if the are only used once?

File: lib/compare_task.rb [hunk 2]
```diff
@@ -94,9 +97,13 @@ def render_comparison(description1, description2, scopes, options = {})
...
 
     output = "Compared descriptions are identical.\n" + output if identical && common_scopes
-
+    output += "Following #{Machinery::pluralize(count, singular, plural)} " \
+      "identical in both descriptions: " + identical_scopes.join(",") if !identical_scopes.empty?
```
Comment (thardeck) [inline lib/compare_task.rb]: Even though this is for the interpreter only one line these are two for us so I would prefer `if` before the the output statement.

File: spec/unit/helper_spec.rb [hunk 3]
```diff
@@ -18,6 +18,33 @@
...
+      end
+      it "returns given plural text if the number is zero" do
+        expect(Machinery::pluralize(0, "text")).to eq "texts"
+      end
+      it "returns given plural text if the number is more than one" do
+        expect(Machinery::pluralize(5, "text")).to eq "texts"
+      end
+    end
+
+    context "pluralform is given" do
```
Comment (thardeck) [inline spec/unit/helper_spec.rb]: a "when" is missing

**Comment by thardeck** [comment]
lgtm besides my comment and the last two commits should be merged

**Comment by keberle94** [comment]
Superseeded with commented changes and squashed in #959

**Pull request closed without merging**